src/apis/platformvm/api.ts

```diff
@@ -528,6 +530,36 @@ export class PlatformVMAPI extends JRPCAPI {
     )
     return response.data.result
   }
+  /**
+   * List amounts that can be claimed: validator rewards, expired deposit rewards, active deposit rewards claimable at current time.
+   *
+   * @param addresses An array of addresses as cb58 strings or addresses as {@link https://github.com/feross/buffer|Buffer}s
+   * @param depositTxIDs An array of deposit transactions ids
+   * @param locktime Optional. The locktime field created in the resulting outputs
+   * @param threshold Optional. The number of signatures required to spend the funds in the resultant UTXO
+   *
+   * @returns Promise for an object containing the amounts that can be claimed.
+   */
+  getClaimables = async (
+    addresses: string[],
+    depositTxIDs: string[],
+    locktime: string = undefined,
+    threshold: number = 1
+  ): Promise<GetClaimablesResponse> => {
+    const params: GetClaimablesParams = {
+      threshold,
+      addresses,
+      depositTxIDs
+    }
+    if (typeof locktime !== "undefined") {
+      params.locktime = locktime
+    }
+    const response: RequestResponseData = await this.callMethod(
+      "platform.getClaimables",
+      params
+    )
+    return response.data.result
```

Comment:
Are you sure that response.data.result is properly deserialized into the structure?

Reply:
FYI, we found a bug with Evgenii and this PR will wait for the fix from his side. I will update you as soon as it's fixed in node's code